### PR TITLE
refactor(queue): add WORKER_NUM and QUEUE_SIZE

### PR DESCRIPTION
**Pull request by appleboy**
## 這個 PR 的目的 / Purpose of this PR

1. Add `WORKER_NUM` and rename `N_COMMENT_QUEUE` to `QUEUE_SIZE`
2. refactor all files. (rename or delete unused functionion)

### CODE REVIEW

File: queue/queue.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	"github.com/golang-queue/queue"
+)
+
+const configPrefix = "go-openbbsmiddleware:queue"
+
+var client *queue.Queue
+
+func Start() error {
+	queueSize := configutil.SetIntConfig(configPrefix, "QUEUE_SIZE", 4096)
```
Comment (chhsiao1981) [inline queue/queue.go:16]: I prefer the original queue/config.go queue/config_util.go setup
In that I can easily know what are the configurable global variables.

I found that queue.InitConfig() actually missed in main_init.go
we should add queue.InitConfig in main_init.go

**Review by chhsiao1981 — CHANGES_REQUESTED** [review]
I prefer the original queue/config.go queue/config_util.go setup
In that I can easily know what are the configurable global variables.

Unless I am 100% sure that the global variable is used only within the same file,
I also prefer global variables are CAPITALIZED_UNDERSCORE.
in that global variables are evil, and I can easily identify them as global variables.
(They shouldn't exist unless they are config variables or consts)

I found that queue.InitConfig() actually missed in main_init.go
we should add queue.InitConfig in main_init.go

File: queue/queue.go (hunk 1 above)
Comment (chhsiao1981) [reply queue/queue.go:16]: (We can refer to types/00-config.go, types/config_util.go, types/config.go and see how the config-variables are setup in this project～)

Comment (appleboy) [reply queue/queue.go:16]: @chhsiao1981 revert the queue/config.go and queue/config_util.go file

Comment (chhsiao1981) [reply queue/queue.go:16]: hi @appleboy 

Thanks for your reply～
I saw your comments～
but I didn't see the changes yet～

Comment (appleboy) [reply queue/queue.go]: @chhsiao1981 I forgot push changes. XD

Comment (appleboy) [reply queue/queue.go]: @chhsiao1981 All done.

**Review by chhsiao1981 — APPROVED** [review]
/lgtm

File: queue/queue.go (hunk 1 above)
Comment (chhsiao1981) [reply queue/queue.go]: /lgtm

**Pull request merged**